AtB-AS/mittatb-app
# fix: handling of app scheme for nfk and atb

Fixes https://github.com/AtB-AS/kundevendt/issues/1920

Came across this issue when testing for https://github.com/AtB-AS/kundevendt/issues/1776 and https://github.com/AtB-AS/kundevendt/issues/2114.

Sets up:

### AtB
```
# Dev
atb-dev://

# Staging
atb-staging://

# Production
atb://
```

### NFK

```
# Dev
nfk-dev://

# Staging
nfk-staging://

# Production
nfk://
```

Based on `APP_SCHEME` in env-folders. This allow us to override based on env and org and easily add support for other orgs.

[See proposed docs](https://github.com/AtB-AS/mittatb-app/blob/b5c73ddddbce5e2c879f33d641ef84444d29b443/docs/OrgConfig.md)

---


Tested on:



This can be a big issue if anyone has both Reis and AtB app running.

---


## How to test locally:

Make sure you have the latest changes and run setup: E.g. `yarn setup dev atb`

```
npx uri-scheme open 'atb-dev://ticketing?transaction_id=321321&payment_id=321321' --android
```

```
npx uri-scheme open 'atb-dev://ticketing?transaction_id=321321&payment_id=321321' --ios
```


## How to test in staging

Completing a ticket purchase should redirect back to active tickets. Using a proxy it should say something like `atb-staging://...`
